backend/sirius-web-emf/src/main/java/org/eclipse/sirius/web/emf/compatibility/modeloperations/CreateViewOperationHandler.java

```diff
@@ -0,0 +1,114 @@
+            CreateView createView) {
+        this.objectService = Objects.requireNonNull(objectService);
+        this.identifierProvider = Objects.requireNonNull(identifierProvider);
+        this.interpreter = Objects.requireNonNull(interpreter);
+        this.childModelOperationHandler = Objects.requireNonNull(childModelOperationHandler);
+        this.createView = Objects.requireNonNull(createView);
+    }
+
+    @Override
+    public IStatus handle(Map<String, Object> variables) {
+        String containerViewExpression = this.createView.getContainerViewExpression();
+
+        var optionalParentElementId = this.interpreter.evaluateExpression(variables, containerViewExpression).asObject().flatMap(parentElement -> {
+            Optional<UUID> optionalElementId = Optional.empty();
+            if (parentElement instanceof Diagram) {
+                Diagram diagram = (Diagram) parentElement;
+                optionalElementId = Optional.of(diagram.getId());
+            } else if (parentElement instanceof Node) {
+                Node node = (Node) parentElement;
+                optionalElementId = Optional.of(node.getId());
+            }
+            return optionalElementId;
+        });
+
+        if (optionalParentElementId.isEmpty()) {
+            return new Failure(""); //$NON-NLS-1$
+        }
+
+        DiagramElementMapping diagramElementMapping = this.createView.getMapping();
+        String diagramElementMappingId = this.identifierProvider.getIdentifier(diagramElementMapping);
+        String targetObjectId = this.objectService.getId(variables.get(VariableManager.SELF));
+
+        // @formatter:off
+        try {
+            UUID descriptionId = UUID.fromString(diagramElementMappingId);
+            ViewCreationRequest viewCreationRequest = ViewCreationRequest.newViewCreationRequest()
+                    .parentElementId(optionalParentElementId.get())
+                    .descriptionId(descriptionId)
+                    .targetObjectId(targetObjectId)
+                    .build();
+
+            Optional.ofNullable(variables.get(IDiagramContext.DIAGRAM_CONTEXT))
+            .filter(IDiagramContext.class::isInstance)
+            .map(IDiagramContext.class::cast)
+            .ifPresent(diagramContext -> diagramContext.getViewCreationRequests().add(viewCreationRequest));
+        } catch (IllegalArgumentException exception) {
+            this.logger.warn(exception.getMessage(), exception);
+        }
+        // @formatter:on
+
+        Map<String, Object> childVariables = new HashMap<>(variables);
+        List<ModelOperation> subModelOperations = this.createView.getSubModelOperations();
+        return this.childModelOperationHandler.handle(this.objectService, this.identifierProvider, this.interpreter, childVariables, subModelOperations);
```

Comment:
If we can't perform the `createView` operation, we continue without issues but if we can't perform the `deleteView` operation, we stop everything. I suspect that this difference in behavior will create an issue someday.